### ISSUE TITLE
Firefox console shows "Attempt to postMessage on disconnected port" from Apollo extension

### ISSUE THREAD

**Issue by swrobel** [apollographql/apollo-client-devtools#1007]
**Intended outcome:**
No extraneous errors are shown in Firefox browser console

**Actual outcome:**
Every browser tab for a site that doesn't use Apollo (ex: github) shows this in the console:
```
Error: Attempt to postMessage on disconnected port [tab.js:1:3247](moz-extension://8d7e5e49-00fa-624f-b1ba-d10fd33b0a05/tab.js)
    sendPortMessage resource://gre/modules/ExtensionChild.jsm:312
```

**How to reproduce the issue:**

Load any page that doesn't use Apollo (this page will do) in Firefox with the Apollo Client Devtools extension installed

**Desktop (please complete the following information):**
 - OS: macOS 13.2.1
 - Browser: Firefox
 - Browser version: 111.0.1
 - Extension version 4.1.4

**Comment by abarreir** [comment]
I've further investigated this issue, and the extension panel stops working after some idle time 100% of the time. A similar issue happened to vuejs devtools https://github.com/vuejs/devtools/issues/1974, making the background script `persistent` in the manifest file fixed the issue.

I've tested the fix locally and can confirm the content script stops losing connection to the background script. Please note that the issue is not reproducible when inspecting the background script with firefox devtools, since this prevents the script from going idle :

<img width="730" alt="bg-script-devtool-msg" src="https://github.com/apollographql/apollo-client-devtools/assets/1942346/9d1e62dd-cc8e-4c21-9f71-54a5280cb0ea">

**Comment by jerelmiller** [comment]
Hey all 👋 

I'd love to revisit this and see if this is still an issue. We introduced a change some time ago that will [automatically reconnect ports when they disconnect](https://github.com/apollographql/apollo-client-devtools/pull/1390) which should help when the background script goes idle due to inactivity. This should be available in [v4.14.2](https://github.com/apollographql/apollo-client-devtools/releases/tag/v4.14.2) or later.

Let me know if you're still seeing this issue, otherwise I'll plan to close this 🙂

**Comment by swrobel** [comment]
Not seeing this anymore. Thanks @jerelmiller!!!
